### PR TITLE
rename hoh operator

### PR DESCRIPTION
**Pull request by morvencao**
ref: https://github.com/stolostron/backlog/issues/24609

Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

File: operator/config/crd/bases/hubofhubs.open-cluster-management.io_multiclusterglobalhubs.yaml [hunk 1]
```diff
@@ -5,20 +5,21 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.8.0
   creationTimestamp: null
-  name: configs.hubofhubs.open-cluster-management.io
+  name: multiclusterglobalhubs.hubofhubs.open-cluster-management.io
 spec:
   group: hubofhubs.open-cluster-management.io
```
Comment (clyang82) [inline operator/config/crd/bases/hubofhubs.open-cluster-management.io_multiclusterglobalhubs.yaml:10]: suggest to use `operator.open-cluster-management.io` as group name.

**Review by clyang82 — CHANGES_REQUESTED** [review]

**Comment by clyang82** [comment]
fix the code smell. most of them are `which is greater than 120 authorized`. I am looking for a tool to configure in vscode to avoid such error.

File: operator/config/crd/bases/hubofhubs.open-cluster-management.io_multiclusterglobalhubs.yaml (hunk 1 above)
Comment (morvencao) [reply operator/config/crd/bases/hubofhubs.open-cluster-management.io_multiclusterglobalhubs.yaml]: done

**Pull request merged**